scanpy/tests/test_paga.py

```diff
@@ -24,7 +24,7 @@
 def pbmc():
     pbmc = pbmc68k_reduced()
     sc.tl.paga(pbmc, groups='bulk_labels')
-    pbmc.obs['cool_feature'] = pbmc[:, 'CST3'].X.squeeze()
+    pbmc.obs['cool_feature'] = pbmc[:, 'CST3'].X.squeeze().copy()
```

Comment:
`pbmc[:, 'CST3'].X.squeeze()` returns an `ArrayView`, which ends up all `[nan, nan, …]`.

Tests fail because I assume matplotlib doesn’t like that anymore.